### DISCUSSION TITLE
[gatsby-theme-minimal-blog]: The banner field doesn't accept/handle GIFs

### DISCUSSION THREAD

**Discussion by suryarajendhran** [LekoArts/gatsby-themes#638] [Questions]
When I try to use a GIF for the banner, I encounter the following error:

```
12:53:15.872 | [2K[1A[2K[Gwarn You can't use childImageSharp together with cartman-happy.gif — use
-- | --
12:53:15.872 | publicURL instead. The childImageSharp portion of the query in this file will
12:53:15.872 | return null:
```

How can I get around this?

**Comment by LekoArts** [comment]
Hi!

Please see: https://github.com/LekoArts/gatsby-themes/discussions/637

So `banner` is only used for `og:image`. Additionally, `gatsby-plugin-sharp` can't handle gifs. If you want to place a gif inside your post, do it directly in the MDX:

```mdx
![Your alt text](./path-to-gif.gif)
```
